Styles/mobileMenu.css

```diff
@@ -0,0 +1,42 @@
+.mobileMenuDiv {
+  background-color: white;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: 999;
+}
+
+.mobileMenuList {
+  list-style: none;
```

Comment:
- [ Optional] You did a god job working on the mobile menu. I just recommend add a little more margin between elements to match exactly the Figma design and to look more professional.